src/main/java/io/reactivex/rxjava3/core/Scheduler.java

```diff
@@ -88,6 +89,28 @@
  * All methods on the {@code Scheduler} and {@code Worker} classes should be thread safe.
  */
 public abstract class Scheduler {
+    static final boolean IS_DRIFT_USE_NANOTIME = Boolean.getBoolean("rx3.scheduler.drift-use-nanotime");
```

Comment:
This can't be tested because the class gets initialized much earlier. Please drop the `final`. Also please document the field with Javadocs.